.cow.json

```
@@ -19,7 +19,13 @@
   "upgrade-only": [
     "dnadesign/silverstripe-elemental-subsites",
     "dnadesign/silverstripe-elemental-userforms",
-    "undefinedoffset/sortablegridfield",
-    "silverstripe/subsites"
+    "silverstripe/lumberjack",
+    "silverstripe/subsites",
+    "symbiote/silverstripe-gridfieldextensions",
+    "symbiote/silverstripe-multivaluefield",
+    "tractorcow/classproxy",
+    "tractorcow/silverstripe-fluent",
+    "tractorcow/silverstripe-proxy-db",
+    "undefinedoffset/sortablegridfield"
```

A lot of these modules aren't actually included in the list of modules Cow attempts to release anyway, so we can shorten this significantly.

```suggestion
    "silverstripe/subsites",
    "tractorcow/silverstripe-fluent",
```